src/engine/enginemaster.cpp

```diff
@@ -121,10 +131,16 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
     // Allocate buffers
     m_pHead = SampleUtil::alloc(MAX_BUFFER_LEN);
     m_pMaster = SampleUtil::alloc(MAX_BUFFER_LEN);
+    m_pBooth = SampleUtil::alloc(MAX_BUFFER_LEN);
     m_pTalkover = SampleUtil::alloc(MAX_BUFFER_LEN);
+    m_pTalkoverHeadphones = SampleUtil::alloc(MAX_BUFFER_LEN);
+    m_pSidechain = SampleUtil::alloc(MAX_BUFFER_LEN);
```

Comment:
According to my "beautiful" drawing, we three buffers to hold the output:
m_pHead, pMaster and pBooth. We may need a MasterMixxx buffer as well, but I think we can reuse pBooth for this.
pMaster can be used for mixing the talkover Mixxx first.